src/protean/core/field/__init__.py

```diff
@@ -110,3 +123,29 @@ def validate(self, value: Any):
         self._run_validators(value)
 
         self.value = value
+
+
+class String(Field):
```

Comment:
Create a separate file for this class, please.

Reply:
Okay this causes a bit of issues as we want uses to be able to import String from Field. In order to do this I will need to move the Field to a base.py and then add String to __all__ in __init__.py.. What do you think

Reply:
Would this structure work?
`src -> core -> fields -> field.py -> Field class`
`src -> core -> fields -> string.py -> String class`

Then we can include in init.py approapriately to have:
`from protean.core.fields import Field, String`